src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -949,11 +948,6 @@ void Model_NodesCleared()
         {
             lock (Nodes)
             {
-                foreach (var nodeViewModel in Nodes)
-                {
-                    this.unsubscribeNodeEvents(nodeViewModel);
-                    nodeViewModel.Dispose();
-                }
```

Comment:
I think I missed your explanation during the tech sharing. Can you explain again why you removed this?